deploy/heroku-deploy/Dockerfile

```diff
@@ -13,6 +13,20 @@ RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-${
 # see installation instructions: https://devcenter.heroku.com/articles/heroku-cli#standalone-installation
 RUN curl https://cli-assets.heroku.com/install-standalone.sh | sh
 
+# Install AWS CLI
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+
+# Verify integrity AWS CLI .zip file (as an additional security check).
+# GPG throws an error if verification fails
+ADD aws_cli_public_pgp_key.txt .
```

Comment:
I think it's [recommended](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) to use `COPY` in this sort of situation to avoid the extra magic that `ADD` tosses in there. Make sense to change it?